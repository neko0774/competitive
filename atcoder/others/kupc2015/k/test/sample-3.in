2
100 0 99 1
47 85 1 1
