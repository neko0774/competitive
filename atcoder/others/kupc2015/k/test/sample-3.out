0.584414703195711
